assume/strategies/flexable.py

```diff
@@ -340,8 +340,8 @@ def calculate_bids(
             min_power[start] = unit.calculate_ramp(
                 op_time, previous_power, min_power[start], current_power
             )
-            bid_quantity = min_power[start] - previous_power
-            if bid_quantity >= 0:
+            bid_quantity = previous_power - min_power[start]
```

Comment:
Just a reminder: Now that the bid_quantity is positive, we have to be careful with the dispatch if product_type = energy_neg, because we then have to subtract this part. (Not implemented yet)

Reply:
For me it is unclear why it is is needed to switch to positive here?
Neither the Commit message, nor the PR state this?
Shouldn't this be separated?

Reply:
yes, true, this is a bug, but the problem is bigger, we need to fix it in a separate commit.